src/command_modules/azure-cli-servicefabric/azure/cli/command_modules/servicefabric/tests/test_sf_commands.py

```diff
@@ -0,0 +1,87 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import unittest
+import os
+
+from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer
+from azure.cli.testsdk import JMESPathCheck as JMESPathCheckV2
+from azure.cli.testsdk.vcr_test_base import (ResourceGroupVCRTestBase,
+                                             JMESPathCheck, NoneCheck)
+
+class ServiceFabricNewClusterTest(ResourceGroupVCRTestBase):
```

Comment:
This is an older test base--these should really use the ScenarioTest base. Don't change them for this PR, but FYI for the future.